glusterapi/device.py

```diff
@@ -0,0 +1,44 @@
+import httplib
+import json
+
+from glusterapi.common import BaseAPI, validate_uuid
+from glusterapi.exceptions import GlusterApiInvalidInputs
+
+
+class DeviceApis(BaseAPI):
+    def device_add(self, peerid="", device=""):
+        """
```

Comment:
Any reason of having peerid and device optional with default empty strings? Since we use them anyway.